### PR TITLE
Adds in new clothing options (cosmetic pr)

### PR DESCRIPTION
**Pull request by Kmc2000**
### Intent of your Pull Request

Adds in a variety of new clothes, and fixes up the shitty shoes I made so that they're slightly(tm) less horrifyingly out of place.

All these items can be found in autodrobes EXCEPT the ferrari, which is admin-only, search for track jacket and trainer and they'll pop up.

I also added a ferrari for fun

PLEASE NOTE:

https://drive.google.com/file/d/0B9VUbibptgF2dW5ta3g4VmRqMnc/view?usp=sharing

The showcase video has some bugged stuff and some sprites that just didn't in it, that's all been fixed for this PR.

Example outfits:

https://i.gyazo.com/67fb79610d03d93c1ed4d9f94fce9c90.mp4

#### Changelog

:cl: Kmc2000
rscadd: Added in some new clothes! check them out at the autodrobe
rscadd: Added an admin only sports car/ ferrari.
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
i should ban you from ftl for this

**Comment by matskuman5** [comment]
Clothes look fine

Ferrari is stupid but since it's admin only it's ok

Shoes look stupid as always and do we really need references to 2010's fashion that will be out of date in 3.4 seconds

**Comment by Jc1469** [comment]
add ferrari to autodrobe tbh

**Comment by Super3222** [comment]
England is my city

**Comment by Oakboscage** [comment]
Please stop adding these fucking sneakers.

**Comment by Ratvar** [comment]
add fereari themed sneakers and sneaker-themed ferrari

**Comment by AshCorr** [comment]
cOsMEtIc pR

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -558,3 +558,33 @@
...
+	desc = "A sporty coat that can be unzipped."
+	icon_state = "trackjacket"
+	item_state = "trackjacket"
+	blood_overlay_type = "coat"
+	body_parts_covered = CHEST|ARMS
+	togglename = "zipper"
+
+/obj/item/clothing/suit/toggle/trackjacket/white
+	name = "blue and white track jacket"
+	desc = "A sporty coat that can be unzipped, it features blue accents."
```
Comment (AshCorr) [inline code/modules/clothing/suits/miscellaneous.dm]: Using periods at the end of the description here but not at the end of the description on sneakers.

**Review by AshCorr — CHANGES_REQUESTED** [review]

File: code/modules/clothing/shoes/colour.dm [hunk 2]
```diff
@@ -107,6 +107,25 @@
 	desc = "Whilst still uncomfortably expensive, these black lowkey trainers make for a great entry into the sneaker game."
 
 
+/obj/item/clothing/shoes/sneakers/yeezy/zebra
+	name = "black and white striped trainers"
+	icon_state = "yeezy_white"
+	item_color = "yeezy_white"
+	desc = "They're SO BRIGHT!!!! AAAAHHHHHHH"
+
```
Comment (AshCorr) [inline code/modules/clothing/shoes/colour.dm:115]: EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE I swear im just going to make a script to find all the extra newlines and remove them...

File: code/modules/clothing/suits/miscellaneous.dm (hunk 1 above)
Comment (Kmc2000) [reply code/modules/clothing/suits/miscellaneous.dm]: fixed

File: code/modules/clothing/head/soft_caps.dm [hunk 3]
```diff
@@ -66,6 +66,13 @@
 	item_color = "green"
 	dog_fashion = null
 
+/obj/item/clothing/head/soft/superb
+	name = "superb square logo cap"
+	desc = "A baseball hat with the word superb on the front, this one word is apparently worth 200 credits shipped."
```
Comment (matskuman5) [inline code/modules/clothing/head/soft_caps.dm]: A baseball hat with the word "superb" on the front. This one word is apparently worth 200 credits shipped.

File: code/modules/clothing/shoes/colour.dm [hunk 4]
```diff
@@ -107,12 +107,31 @@
...
+	name = "khaki trainers"
+	icon_state = "yeezy_green"
+	item_color = "yeezy_green"
+	desc = "Slightly more practical trainers that don't glow in the dark."
+
+/obj/item/clothing/shoes/sneakers/yeezy/red
+	name = "black and red trainers"
+	icon_state = "yeezy_red"
+	item_color = "yeezy_red"
+	desc = "Wait a minute these look...ehh probably not, the syndicate only wear combat boots."
```
Comment (matskuman5) [inline code/modules/clothing/shoes/colour.dm]: Wait a minute, these look... eh, probably not. The syndicate only wear combat boots.

File: code/modules/clothing/shoes/colour.dm [hunk 5]
```diff
@@ -107,12 +107,31 @@
...
+
 
 /obj/item/clothing/shoes/sneakers/nmd
 	name = "primeknit trainers"
 	icon_state = "nmd_pk"
 	item_color = "nmd_pk"
-	desc = "Stylish running shoes, the comfy soles give you a spring in your step"
+	desc = "Stylish running shoes, the comfy soles give you a spring in your step."
```
Comment (matskuman5) [inline code/modules/clothing/shoes/colour.dm]: Stylish running shoes. The comfy soles give you a spring to your step.

File: code/modules/clothing/suits/miscellaneous.dm (hunk 1 above)
Comment (matskuman5) [inline code/modules/clothing/suits/miscellaneous.dm]: A sporty coat that can be unzipped. It features blue accents.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 6]
```diff
@@ -558,3 +558,33 @@
...
+	icon_state = "trackjacket_white"
+	item_state = "trackjacket_white"
+
+/obj/item/clothing/suit/toggle/trackjacket/paint
+	name = "paint splat track jacket"
+	desc = "A track jacket that has a paint splat design on it, it looks kinda like a galaxy, neat."
```
Comment (matskuman5) [inline code/modules/clothing/suits/miscellaneous.dm]: A track jacket that has a paint splat design on it. It looks kinda like a galaxy, neat.

File: code/modules/vehicles/sportscar.dm [hunk 7]
```diff
@@ -0,0 +1,51 @@
+/obj/vehicle/sportscar
+	name = "Nanotrasen 558 SE sport"
+	desc = "A beautiful car, merging the modern nanotrasen ideals with the classic Italian sports car aesthetic, this thing goes F A S T"
```
Comment (matskuman5) [inline code/modules/vehicles/sportscar.dm]: A beautiful car, merging the modern NanoTrasen ideals with the classic Italian sports car aesthetic. 
This thing goes F A S T!

File: code/modules/clothing/head/soft_caps.dm (hunk 3 above)
Comment (matskuman5) [reply code/modules/clothing/head/soft_caps.dm]: References to real life stuff like "Superb" doesn't fit the theme of SS13 at all.

**Comment by ghost** [comment]
theres not 20 lines of whitespace between every object you added 2/10

**Review by AshCorr — APPROVED** [review]

File: code/modules/clothing/shoes/colour.dm [hunk 8]
```diff
@@ -107,12 +107,31 @@
...
+	item_color = "yeezy_red"
+	desc = "Who said that syndicate operatives only wear combat boots?"
+
 
 /obj/item/clothing/shoes/sneakers/nmd
 	name = "primeknit trainers"
 	icon_state = "nmd_pk"
 	item_color = "nmd_pk"
-	desc = "Stylish running shoes, the comfy soles give you a spring in your step"
+	desc = "Stylish running shoes.The comfy soles give you a spring in your step."
```
Comment (Amelia0010) [inline code/modules/clothing/shoes/colour.dm]: What even are spaces

File: code/modules/vehicles/sportscar.dm [hunk 9]
```diff
@@ -0,0 +1,51 @@
+/obj/vehicle/sportscar
+	name = "Nanotrasen 558 SE sport"
+	desc = "A beautiful car, merging the modern NanoTrasen ideals with the classic Italian sports car aesthetic.This thing goes F A S T!"
```
Comment (Amelia0010) [inline code/modules/vehicles/sportscar.dm]: Space, the final frontier

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
Remember to add spaces after dots

**Comment by Jc1469** [comment]
ra ra rasputin still missing space it seems

`aesthetic.This`

File: code/modules/vehicles/sportscar.dm [hunk 10]
```diff
@@ -1,6 +1,6 @@
 /obj/vehicle/sportscar
 	name = "Nanotrasen 558 SE sport"
-	desc = "A beautiful car, merging the modern NanoTrasen ideals with the classic Italian sports car aesthetic.This thing goes F A S T!"
+	desc = "A beautiful car,merging the modern NanoTrasen ideals with the classic Italian sports car aesthetic.This thing goes F A S T!"
```
Comment (Amelia0010) [inline code/modules/vehicles/sportscar.dm]: you actually removed a space from somewhere it should be, instead of adding a space that's needed. :thailand:

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
*sigh*

File: code/modules/vehicles/sportscar.dm (hunk 9 above)
Comment (Amelia0010) [reply code/modules/vehicles/sportscar.dm]: you still haven't put a space after the bloody dot, god dammit KMC

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
why...

**Comment by Kmc2000** [comment]
WOOHOO 🎉

**Pull request merged**